core/include/drivers/atmel_rstc.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2021, Microchip
+ */
+#ifndef ATMEL_RSTC_H
```

Comment:
Please add a `__DRIVERS_` prefix. The `__` part is optional, but I prefer using it.